SPECS/xorg-x11-util-macros/xorg-x11-util-macros.spec

```diff
@@ -0,0 +1,243 @@
+%global pkgname util-macros
+%global debug_package %{nil}
+Summary:        X.Org X11 Autotools macros
+Name:           xorg-x11-util-macros
+Version:        1.19.2
+Release:        8%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.x.org
+Source0:        https://www.x.org/pub/individual/util/%{pkgname}-%{version}.tar.bz2
+
+Provides:       pkgconfig(xorg-macros) = %{version}-%{release}
```

Comment:
Can we please move the `Provides` below all the `BuildRequires/Requires`

Reply:
Sure.